client/src/model/geo_slice.ts

```diff
@@ -0,0 +1,54 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { ApiClient } from '@/api/api_client';
+import { GeoIdentifiers } from '@/model/geo_identifiers';
+import { AppDispatch } from '@/model/store';
+
+/**
+ * State of geo selection.
+ */
+interface GeoState {
+  geoids?: GeoIdentifiers;
+}
+
+const initialState: GeoState = {};
+const client = new ApiClient();
+
+const geoSlice = createSlice({
```

Comment:
can you link documentation for createSlice here?

Reply:
Done!